# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1474380895 14400
#      Tue Sep 20 10:14:55 2016 -0400
# Node ID 9d50627498141d150755d18ab8e71a3220e40147
# Parent  34f98c1e4d568a59e6aecb2eb6abe0a9a5c34419
maint: Bump version number for 4.2.0-rc2.

* configure.ac: Bump version to 4.2.0-rc2.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -14,31 +14,31 @@ dnl Copyright (C) 1993-2016 John W. Eato
 ### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 ### for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <http://www.gnu.org/licenses/>.
 
 AC_PREREQ([2.63])
-AC_INIT([GNU Octave], [4.2.0-rc1], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [4.2.0-rc2], [http://octave.org/bugs.html], [octave])
 
 dnl Note that the version number is duplicated here and in AC_INIT
 dnl because AC_INIT requires it to be static, not computed from
 dnl shell variables.
 OCTAVE_MAJOR_VERSION=4
 OCTAVE_MINOR_VERSION=2
-OCTAVE_PATCH_VERSION=0-rc1
+OCTAVE_PATCH_VERSION=0-rc2
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2016 John W. Eaton and others."
 
-OCTAVE_RELEASE_DATE="2016-09-07"
+OCTAVE_RELEASE_DATE="2016-09-20"
 
 ## The "API version" is used as a way of checking that interfaces in the
 ## liboctave and libinterp libraries haven't changed in a backwardly
 ## incompatible way when loading .oct files.  A better way to do that is
 ## with library versioning, but not all systems support that.
 ## NOTE: This macro will be removed in a future version of Octave.  If
 ## you insist on checking for features using a version number, use the
 ## OCTAVE_MAJOR_VERSION, OCTAVE_MINOR_VERSION, and
